pages/contracts/[wallet]/[contractName]/[version].tsx

```diff
@@ -109,18 +110,22 @@ const ContractsNamePageWrapped = () => {
   );
 };
 
-export default function ContractNamePage() {
+const ContractNamePage: NextPageWithLayout = function () {
   return (
     <PublisherSDKContext>
       <ContractsNamePageWrapped />
     </PublisherSDKContext>
   );
-}
+};
 
 ContractNamePage.getLayout = function getLayout(page: ReactElement) {
   return <AppLayout>{page}</AppLayout>;
 };
 
+ContractNamePage.trackingScope = "released_contract_detail";
```

Comment:
```suggestion
ContractNamePage.trackingScope = "released_contract_detail";
```

this one makes sense to me, I was calling it `specific_release` but every page is gonna be a specific release